documentation/cloud-manager-guide/environments/environment-details/environment-overview.md

```diff
@@ -0,0 +1,145 @@
+---
+keywords:
+title: Environment Overview
+description: Components and navigation for an environment's detailed view
+---
+
+# Environment Overview
```

Comment:
Maybe we call this something like, "Environment Details" so as not to be too confused with the Environments Overview that we already have overviewing the entire environments area?